astrovascpy/typing.py

```diff
@@ -23,9 +23,49 @@ class VasculatureAxis(enum.IntEnum):
     Z = 2
 
 
-class VasculatureParams(typing.TypedDict):
+class VasculatureParams(TypedDict):
+    """
+    Required parameters used in the model:
+
+    |  vasc_axis: vasculature axis corresponding to x, y, or z. Should be set to 0, 1, or 2.
+    |  depth_ratio: depth along the vasc_axis. This is the portion of the vasculature where there are inputs.
+    |  max_nb_inputs: maximum number of entry nodes where we inject the flow into the vasculature. Should be >= 1.
+    |  blood_viscosity: plasma viscosity in g.µm^-1.s^-1
+    |  base_pressure: reference pressure in g * um^{-1} * s^{-2}. At resting state equal to the external pressure
+
+    Stochastic simulation parameters (optional):
+
+    |  threshold_r: radius (µm) threshold. A radius smaller than the threshold is considered a capillary. A radius bigger than the threshold is considered an artery.
+
+    |  c_cap: constant used in the ROU parameter calibration for capillaries
+    |  c_art: constant used in the ROU parameter calibration for arteries
+
+    |  max_r_capill: max radius change factor for capillaries
+    |  t_2_max_capill: time (in seconds) to reach r_max from 0 for capillaries
+    |  max_r_artery: max radius change factor for arteries
+    |  t_2_max_artery: time (in seconds) to reach r_max from 0 for arteries
+
+    PETSc Linear solver parameters (optional):
+
+    |  solver: iterative linear solver used by PETSc
+    |  max_it: maximum number of solver iterations
+    |  r_tol: relative tolerance
```

Comment:
This renders like this:
![image](https://github.com/BlueBrain/AstroVascPy/assets/2151261/7664a266-5e6a-4263-ab76-194f182391ca)

But ideally, we'd want them to look like this:
![image](https://github.com/BlueBrain/AstroVascPy/assets/2151261/e08b2143-bdd1-4131-b804-7426d48650ff)

Maybe something like this would be better?
```suggestion
    Required parameters used in the model are as follows:

    Args:
        vasc_axis: vasculature axis corresponding to x, y, or z. Should be set to 0, 1, or 2.
        depth_ratio: depth along the vasc_axis. This is the portion of the vasculature where there are inputs.
        max_nb_inputs: maximum number of entry nodes where we inject the flow into the vasculature. Should be >= 1.
        blood_viscosity: plasma viscosity in g.µm^-1.s^-1
        base_pressure: reference pressure in g * um^{-1} * s^{-2}. At resting state equal to the external pressure

    Optional stochastic simulation parameters can also be provided.

    Args:
        threshold_r: radius (µm) threshold. A radius smaller than the threshold is considered a capillary. A radius bigger than the threshold is considered an artery.

        c_cap: constant used in the ROU parameter calibration for capillaries
        c_art: constant used in the ROU parameter calibration for arteries

        max_r_capill: max radius change factor for capillaries
        t_2_max_capill: time (in seconds) to reach r_max from 0 for capillaries
        max_r_artery: max radius change factor for arteries
        t_2_max_artery: time (in seconds) to reach r_max from 0 for arteries

    Optional PETSc Linear solver parameters.

    Args:
        solver: iterative linear solver used by PETSc
        max_it: maximum number of solver iterations
        r_tol: relative tolerance
```
I'm not sure if it works with multiple `Args` things, but it would be worth a try IMO.

Reply:
thanks! Now I try!